9.3-jre8/alpine/Dockerfile

```diff
@@ -44,9 +44,10 @@ RUN set -xe \
 	&& modules="$(grep -- ^--module= "$JETTY_HOME/start.ini" | cut -d= -f2 | paste -d, -s)" \
 	&& java -jar "$JETTY_HOME/start.jar" --add-to-startd="$modules,setuid" \
 
-	# Remove installed packages
+	# Remove installed packages and various cleanup
 	&& apk del .build-deps \
-	&& rm -fr .build-deps
+	&& rm -fr .build-deps \
+	&& rm -rf /tmp/hsperfdata_root /etc/group- /etc/passwd- /etc/shadow-
```

Comment:
Deleting the `/etc/group-` and other file files in this layer isn't ideal. They need to be deleted in the same layer they're added, which is the `adduser` layer above 👍